### ISSUE TITLE
Problem with a DW11 2GIG/Vivint Window Sensor

### ISSUE THREAD

**Issue by Sleepybear** [fusterjj/HoneywellSecurityMQTT#9]
I honestly can't figure this out. 

I have 8 sensors all working perfectly with this app, so thank you for that!

However, we have 1 2GIG DW11 from a Vivint system that is no longer active, that doesn't want to play nice. It shows as Invalid Payloads in the log, but then in Home Assistant it randomly picks up a trigger of open/close every few hours.

**Comment by issvc** [comment]
Hello. I see your very recent post. It looks like you successfully installed the component. I attempted to install it via HACS in HASS.IO, but it failed with 'Repository structure for master is not compliant' error. Please let me know if there is another way to install this custom component. Thank you in advance!

**Comment by Sleepybear** [comment]
> Hello. I see your very recent post. It looks like you successfully installed the component. I attempted to install it via HACS in HASS.IO, but it failed with 'Repository structure for master is not compliant' error. Please let me know if there is another way to install this custom component. Thank you in advance!

This is a stand alone binary that broadcasts over MQTT to Home Assistant, it's not installed as an addon.

**Comment by jonpage88** [comment]
Did you ever get this figured out? I am having the same issue.

**Comment by Sleepybear** [comment]
Nope, haven't been able to get this figured out. There is a second sensor that also always triggers as open for no apparent reason. After a HA reboot, it will show closed for like 5 mins, then randomly open and stay open.

**Comment by guttermonk** [comment]
#https://github.com/merbanan/rtl_433/issues/1261
